parse_pom_sequence.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#
+# Common utilities used by tools for automating tagging and release
+#
+
+#We want any failures to be fatal
+# Depends: maven
+
+set -e
+
+if ! `which mvn 1>/dev/null 2>&1` ; then
+  echo "Maven dependency not located"
+  exit 2
+fi
+
+# overwrite versionfile to pom.xml
```

Comment:
I see the term overwrite, but I'm not sure of the inheritance model in bash, are we actually inheriting from another bash? if so, do we need to declare this fact?

Reply:
We're basically just overwriting function definitions from the parse_cmake.sh file that gets sourced earlier by go_rc or go_gold.  The new functions keep the same parameters and output behavior, but are specific to working on a pom.  Only gets sourced IF the pom is present in the local directory we're working from (which is how Richard does his workflow, his command cd's into the local directory and commands from that location)